src/main/java/it/gov/pagopa/pu/bff/service/debt_position/DebtPositionRetrieverServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package it.gov.pagopa.pu.bff.service.debt_position;
+
+import it.gov.pagopa.pu.auth.dto.generated.UserInfo;
+import it.gov.pagopa.pu.bff.connector.debt_position.DebtPositionService;
+import it.gov.pagopa.pu.bff.dto.DebtPositionViewFiltersDTO;
+import it.gov.pagopa.pu.bff.dto.generated.PagedDebtPositionView;
+import it.gov.pagopa.pu.bff.mapper.DebtPositionViewMapper;
+import it.gov.pagopa.pu.bff.service.AuthorizationService;
+import it.gov.pagopa.pu.debtpositions.dto.generated.DebtPositionView.DebtPositionOriginEnum;
+import java.util.List;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DebtPositionRetrieverServiceImpl implements DebtPositionRetrieverService {
+
+  private final DebtPositionService debtPositionService;
+  private final DebtPositionViewMapper debtPositionViewMapper;
+
+  public DebtPositionRetrieverServiceImpl(DebtPositionService debtPositionService,
+    DebtPositionViewMapper debtPositionViewMapper) {
+    this.debtPositionService = debtPositionService;
+    this.debtPositionViewMapper = debtPositionViewMapper;
+  }
+
+  public PagedDebtPositionView getDebtPositionViews(
+    DebtPositionViewFiltersDTO filtersDTO, Pageable pageable, UserInfo loggedUser, String accessToken) {
+    AuthorizationService.isUserEnabledToOrganizationId(filtersDTO.getOrganizationId(), loggedUser);
+    return debtPositionViewMapper.mapToPagedDebtPositionView(
+      debtPositionService.getDebtPositionViews(
+        filtersDTO,
+        buildDebtPositionOriginFilterList(),
+        loggedUser.getMappedExternalUserId(),
+        pageable,
+        accessToken)
+    );
+  }
+
+  private static List<String> buildDebtPositionOriginFilterList() {
```

Comment:
why don't you just create a constant to hold this value instead of creating it each time?